action.yml

```
@@ -121,6 +125,20 @@ runs:
             TWINE_OPTS="$TWINE_OPTS --skip-existing"
           fi
           python -m pip install twine
+          if [[ -z "$TWINE_PASSWORD" ]]; then
+            # vis https://docs.pypi.org/trusted-publishers/using-a-publisher/#the-manual-way
+            export TWINE_PASSWORD=$(
+        python -c 'import json, urllib.request as req
+        audience = req.urlopen("'${INPUT_REGISTRY_DOMAIN}'/_/oidc/audience").read()
+        token = json.load(req.urlopen(req.Request(
+            url="'${ACTIONS_ID_TOKEN_REQUEST_URL}'",
+            data=audience,
+            headers={"Authorization": "bearer '${ACTIONS_ID_TOKEN_REQUEST_TOKEN}'"})))
+        print(json.load(req.urlopen(req.Request(
+            url="https://'${INPUT_REGISTRY_DOMAIN}'/_/oidc/github/mint-token",
```

couldn't see an easy way to use f-strings without nested quote hell. wdyt?

```suggestion
        audience = req.urlopen(os.getenv("INPUT_REGISTRY_DOMAIN") + "/_/oidc/audience").read()
        token = json.load(req.urlopen(req.Request(
            url=os.getenv("ACTIONS_ID_TOKEN_REQUEST_URL"),
            data=audience,
            headers={"Authorization": "bearer " + os.getenv("ACTIONS_ID_TOKEN_REQUEST_TOKEN")})))
        print(json.load(req.urlopen(req.Request(
            url="https://" + os.getenv("INPUT_REGISTRY_DOMAIN") + "/_/oidc/github/mint-token",
```